lib/registries/addon/components/registries-recent-list/template.hbs

```diff
@@ -15,9 +15,9 @@
             <div class='col-xs-12'>
                 <h4>
                     <a
-                        href={{reg.registrationURL}}
+                        href={{reg.url}}
```

Comment:
this'll cause a full reload -- could use `<OsfLink>` to do an in-app transition (and also avoid constructing a url by hand)